apps/worker/src/app/workflow/usecases/send-message/send-message.usecase.ts

```diff
@@ -63,6 +63,12 @@ export class SendMessage {
 
     const stepType = command.step?.template?.type;
 
+    if (!shouldRun.passed || !preferred) {
```

Comment:
as far as I remember we wanted to send that analytic event even if filters did not pass 
cc @scopsy

Reply:
@davidsoderberg i have mis understood your message earlier on Discord. The event should be still send even if the job is canceled. I was thinking you meant the other way in the message. Sorry for the confusion.